dtm0/net.h

```diff
@@ -46,62 +52,124 @@
 struct m0_be_op;
 struct m0_buf;
 
+struct m0_dtm0_net_redo {
```

Comment:
```
/*
 * DTM0 transport overview
 * -----------------------
 *
 *   DTM0 transport is a component responsible for sending and receiving
 * of DTM0 messages (P, REDO, EOL and so on) over Motr RPC layer.
 * Transport contains a fixed amount of free buffers in snd and rcv queues.
 * Because of that, the caller may need to wait until data is processed.
 */
```

Why do we need such "DTM0 transport"?